src/utils/const.js

```diff
@@ -0,0 +1,4 @@
+export const agent1 = {
+  "id" : "HcScIio76M8cbDqhrsZKp37W8YZ4y8X9v9epzbyhRzqpjeb5sigdC4puN6b5koi",
```

Comment:
this is brittle compared to reading it out of the conductor directly. Is it not possible to read it from the conductor anymore?

Reply:
We can read the `confix.nix`?